specs/_features/eip7594/das-core.md

```diff
@@ -289,9 +288,7 @@ The particular columns that a node custodies are selected pseudo-randomly as a f
 
 At each slot, a node needs to be able to readily sample from *any* set of columns. To this end, a node SHOULD find and maintain a set of diverse and reliable peers that can regularly satisfy their sampling demands.
 
-A node runs a background peer discovery process, maintaining at least `TARGET_NUMBER_OF_PEERS` of various custody distributions (both `custody_size` and column assignments). The combination of advertised `custody_size` size and public node-id make this readily and publicly accessible.
-
-`TARGET_NUMBER_OF_PEERS` should be tuned upward in the event of failed sampling.
+A node runs a background peer discovery process, maintaining at least 70 peers of various custody distributions (both `custody_size` and column assignments). The combination of advertised `custody_size` size and public node-id make this readily and publicly accessible. A node may increase its peer count in the event of failed sampling.
```

Comment:
I think there is really no reason to put the number at all. How about just specifying that you should have enough peers to cover all the columns?

Reply:
Okay to me to not recommend a number at all, but would like to the thumbs up of researchers before. The purpose of this PR is only to remove the config variable, not the number itself.

Reply:
I think removing it makes sense. We should wait for @fradamt to comment.

Reply:
Removing the config variable sounds good. As for the rest, I would maybe lean towards having a paragraph somewhere which gives some indications on how to think about the number of peers in relation to sampling, rather than giving a specific number. For example in the spirit of [this discussion](https://github.com/ethereum/consensus-specs/pull/3779#issuecomment-2231109016)

Reply:
However, I think it's okay to make another PR to remove this number and rephrase this paragraph. We can merge it as it is.